src/theme/MuiTheme.ts

```diff
@@ -9,27 +9,29 @@ const NotoSansJP = 'Noto Sans JP, sans-serif'
 const fontWeightRegular = 400
 
 const MuiTheme = createMuiTheme({
+  // Default values at:
+  // https://github.com/mui-org/material-ui/blob/master/packages/material-ui/src/styles/createPalette.js
   palette: {
     // You can enter the main color here https://material-ui.com/style/color/#color-tool
     // and get the associated dark and light colors
     primary: {
+      dark: '#b26500',
+      light: '#ffa733',
+      main: '#FF9100',
+    },
+    secondary: {
       dark: '#000027',
       light: '#333360',
       main: '#010039',
-      contrastText: '#FFFFFF'
-    },
-    secondary: {
-      dark: '#b29500',
-      light: '#ffde33',
-      main: '#FFD600',
     },
     background: {
       default: '#13194F',
       paper: '#000000',
     },
     text: {
       primary: '#FFFFFF'
-    }
+    },
+    type: 'dark'
```

Comment:
Due to this, the form border colors depend on the color pallete type being either 'light' or 'dark'. 

https://github.com/mui-org/material-ui/blob/v3.7.0/packages/material-ui/src/OutlinedInput/OutlinedInput.js#L13

```js
 const borderColor =
    theme.palette.type === 'light' ? 'rgba(0, 0, 0, 0.23)' : 'rgba(255, 255, 255, 0.23)';
```